<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BWMapping</class>
 <widget class="QMainWindow" name="BWMapping">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>374</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>BW Mapping Tool</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>thread_16xLG.png</normaloff>thread_16xLG.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>121</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Map Type</string>
    </property>
    <widget class="QRadioButton" name="partialmap_radio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>82</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Partial</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="fullmap_radio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Full</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="map_connectivity_combo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="map_startpoint_combo">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Map Connectivity</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Partial Map Starting Object</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Extra Information:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generate_pushButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>290</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="BI7_checkBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>BI7 Converted Flows</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="flowVol_checkBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Flow Volumes for Last Run</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="storedRec_checkBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Stored Record Count</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>518</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="fileRegenerate_Database"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="fileRegenerate_Database">
   <property name="text">
    <string>Regenerate Database</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
